src/_pytest/assertion/util.py

```diff
@@ -156,20 +157,30 @@ def has_default_eq(
     return True
 
 
-def assertrepr_compare(config, op: str, left: Any, right: Any) -> Optional[List[str]]:
+def assertrepr_compare(
+    config, op: str, left: Any, right: Any, use_ascii: bool = False
+) -> Optional[List[str]]:
     """Return specialised explanations for some operators/operands."""
     verbose = config.getoption("verbose")
+
+    use_ascii = (
```

Comment:
This would be a good place to leave a comment like *# Strings which normalize equal are often hard to distinguish when printed; use ascii() to make this easier - see issue #3246.*